File: 136-SingleNumber/136-SingleNumber.cpp
// Last updated: 6/27/2025, 2:30:30 AM
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int ans = 0;
        for(int val: nums){
            ans = ans ^ val;
        }
        return ans;
    }
};

File: 136-SingleNumber/136-SingleNumber.java
// Last updated: 6/27/2025, 2:30:34 AM
class Solution {
    public int singleNumber(int[] nums) {
        int unique = 0;
        for(int index = 0; index < nums.length; index++){
            unique ^= nums[index];
        }
        return unique;
    }
}